lambdas/cfn-ssm-param-macro/ssm_param/app.py

```diff
@@ -0,0 +1,35 @@
+import boto3
+import json
+import traceback
+
+def handle_transform(parameters):
+    secure = False
+    if 'Secure' in parameters and parameters['Secure'] is True:
+        secure = True
+    if 'SsmKeyName' not in parameters:
+        raise ValueError('"SsmKeyName" parameter is required')
+    keyname = parameters['SsmKeyName']
+    client = boto3.client('ssm')
+    response = client.get_parameter(
+        Name=keyname,
+        WithDecryption=secure)
+    print(response)
```

Comment:
this may leak secrets. i suggest removing it.

Reply:
oh dear, good find